README.md

```diff
@@ -31,7 +31,7 @@
 
 The Puppet dsc module manages Windows PowerShell DSC (Desired State Configuration) resources.
 
-This module generates Puppet types based on DSC resources' MOF (Managed Object Format) schema files.
+This module generates Puppet types based on DSC Resources MOF (Managed Object Format) schema files.
 
 In this version, the following DSC Resources are already built and ready for use:
 
```

Comment:
@glennsarti Can you say more about why? I ask because when Docs discussed the .x issue, we didn't come away feeling that .x adds anything. To my mind, as a user, Software 5.1 and Software 5.8 are still both Software 5; any issue present in Software 5 is present in .x unless stated otherwise.